pywren/wrenhandler.py

```diff
@@ -25,11 +26,12 @@
     import wrenutil # pylint: disable=relative-import
     import version  # pylint: disable=relative-import
 
-PYTHON_MODULE_PATH = "/tmp/pymodules"
-CONDA_RUNTIME_DIR = "/tmp/condaruntime"
+PYTHON_MODULE_PATH = "/tmp/pymodules_{0}"
+CONDA_RUNTIME_DIR = "/tmp/condaruntime_{0}"
 RUNTIME_LOC = "/tmp/runtimes"
-JOBRUNNER_CONFIG_FILENAME = "/tmp/jobrunner.config.json"
-JOBRUNNER_STATS_FILENAME = "/tmp/jobrunner.stats.txt"
+JOBRUNNER_CONFIG_FILENAME = "/tmp/jobrunner_{0}.config.json"
```

Comment:
So this will get filled by `pid` right ? Can you include that in the comment ? It'll be good to know what is ETag specific and what is PID specific